micro-rdk/src/common/camera/mod.rs

```diff
@@ -1,13 +1,20 @@
-use super::{generic::DoCommand, status::Status};
+use super::{generic::DoCommand, registry::ComponentRegistry, status::Status};
 use bytes::BytesMut;
 use std::sync::{Arc, Mutex};
 use thiserror::Error;
 
-// Enables FakeCamera for native server
-#[cfg(all(feature = "camera", feature = "native", feature = "builtin-components"))]
+#[cfg(feature = "camera")]
 mod fake_camera;
-#[cfg(all(feature = "camera", feature = "native", feature = "builtin-components"))]
-pub(crate) use fake_camera::register_models;
+
+#[allow(unused)]
+pub(crate) fn register_models(registry: &mut ComponentRegistry) {
+    #[cfg(feature = "camera")]
```

Comment:
Yeah I feel like this should still be gated on builtin-components too

Reply:
With the change to the `camera` feature in Cargo.toml in this PR, builtins is active anytime camera is active. Can update this when conclusions are made in the feature dependencies/enablings

Reply:
following offline conversation
- camera: enable traits
- camera && builtin: enable traits, fake_camera, esp32camera (on esp32)

Reply:
done